subsys/bluetooth/host/hci_core.c

```diff
@@ -4734,6 +4734,8 @@ static int set_ad(u16_t hci_op, const struct bt_ad *ad, size_t ad_len)
 				len = 31 - (set_data->len + 2);
 				if (type != BT_DATA_NAME_COMPLETE || !len) {
 					net_buf_unref(buf);
+					BT_ERR(
+						"Advertising data does not fit into it's buffer");
```

Comment:
s/it's/its/

also, why the split into two lines?

Reply:
To satisfy the 80 char limit.

Thanks, will fix.

Reply:
@jhedberg : Fixed. Please re-asses when convenient.

Reply:
Even with the line split it still exceeds 80 characters (unless you have some non-standard tab width - the assumption with the coding style is 8). How about something shorter, yet equally descriptive, e.g.:
```
        BT_ERR("Too big advertising data");
```

Reply:
Fixed.